src/sklearn_evaluation/plot/precision_recall.py

```diff
@@ -19,7 +20,7 @@
 def _set_ax_settings(ax, name):
     ax.set_title(name)
     ax.set_xlim([0.0, 1.0])
-    ax.set_ylim([0.0, 1.05])
+    ax.set_ylim([0.0, 1.0])
```

Comment:
any reason to change this?

Reply:
This adds a small margin at the top. removed this since we don't have an upper border.

Let me know which one is better

![image](https://user-images.githubusercontent.com/103026689/220856203-e58656bc-7ea2-4e46-a4fb-62d5672b93ae.png)

![image](https://user-images.githubusercontent.com/103026689/220856115-8e9b5342-cee9-46b1-a175-3d1715f27848.png)

Reply:
the top one looks better